apps/settings/js/modules/bluetooth/bluetooth_context.js

```diff
@@ -48,11 +48,16 @@ define(function(require) {
     /**
      * State of Bluetooth.
      *
+     * Set the default state as true in bug 1191104 Since Bluetooth
```

Comment:
In current structure, any change under bluetooth/js/modules should be synced with Settings Bluetooth module (apps/settings/js/modules/bluetooth).